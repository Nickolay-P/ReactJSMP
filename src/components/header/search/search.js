import React, { Component } from 'react';

export class Search extends Component {
    render() {
        return (
            <div>
                <div className="rouletteLabel">netflixroulette</div>
                <div className="findLabel">FIND YOUR MOVIE</div>
                <div className="inputField">
                <input type="search" className="siteSearch" name="q" aria-label="Search through site content"></input>   
                <button type="submit" className="arrowButton"></button>
                </div>
                <div className="searchPanel">
                    <div className="searchTitleGenre">
                        <div className="searchLabel">SEARCH BY&nbsp;</div>
                        <button className="titleButton">TITLE</button>
                        <button className="genreButton">GENRE</button>
                    </div>
                    <button className="searchButton">SEARCH</button>
                </div>
            </div>
        );
    }
}