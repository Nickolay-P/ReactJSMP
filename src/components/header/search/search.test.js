

import React from 'react';
import renderer from 'react-test-renderer';
import {Search} from './search.js';


it('Search rendered correctly', () => {
  const AppComponent = renderer.create(<Search />).toJSON();
  expect(AppComponent).toMatchSnapshot();
});