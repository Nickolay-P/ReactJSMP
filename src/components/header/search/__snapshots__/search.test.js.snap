// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search rendered correctly 1`] = `
<div>
  <div
    className="rouletteLabel"
  >
    netflixroulette
  </div>
  <div
    className="findLabel"
  >
    FIND YOUR MOVIE
  </div>
  <div
    className="inputField"
  >
    <input
      aria-label="Search through site content"
      className="siteSearch"
      name="q"
      type="search"
    />
    <button
      className="arrowButton"
      type="submit"
    />
  </div>
  <div
    className="searchPanel"
  >
    <div
      className="searchTitleGenre"
    >
      <div
        className="searchLabel"
      >
        SEARCH BYÂ 
      </div>
      <button
        className="titleButton"
      >
        TITLE
      </button>
      <button
        className="genreButton"
      >
        GENRE
      </button>
    </div>
    <button
      className="searchButton"
    >
      SEARCH
    </button>
  </div>
</div>
`;
